### PR TITLE
Improve confreader 

### PR DESCRIPTION
**Pull request by Nuwini**
Update confreader to record database credentials

## Motivation and Context
#106

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by rehrumesh** [comment]
Hi. Thanks for the contribution. However any sort of credentials should moved into the environment instead of storing in config files. Can you do the necessary changes and append commits to the PR?

File: components/core/main.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{"user": {"username": "", "password": ""},
```
Comment (rehrumesh) [inline components/core/main.conf]: Lets move user credentials out from the config files to the environment variables

File: components/core/ConfReader.py [hunk 2]
```diff
@@ -15,13 +15,14 @@
...
-    dlConf = get_conf_reader('dl.conf')
-    emailConf = get_conf_reader('email.conf')
-
-    if any(val == '' for val in dlConf.values()):
-        sys.exit("Please set all the values in dl.conf")
-    if any(val == '' for val in emailConf.values()):
-        sys.exit("Please set all the values in email.conf")
+    conf = get_conf_reader('main.conf')
+
+    if any(val == '' for val in conf['user'].values()):
```
Comment (rehrumesh) [inline components/core/ConfReader.py]: Assuming credentials will be stored in the env, make changes to read from the env instead from the conf file. See my comment in #106

**Comment by Nuwini** [comment]
I will make the necessary changes.

**Comment by agentmilindu** [comment]
@Nuwini Can you do the requested changes?

**Comment by agentmilindu** [comment]
@Nuwini Can you do the requested changes?

**Comment by kmehant** [comment]
Right now, we have a working confReader on the `develop` branch. I think moving all the configurations to a single file makes sense but seems to be not required right now. Coming to database credentials please look at my comment here https://github.com/scorelab/Bassa/issues/106#issuecomment-498549963  
Please let us know if you have any points regarding this 😃.

**Pull request closed without merging**